src/WebDocs/wwwroot/app/functions/Lockplumber/parameters.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
i have parameters here can ou look at the source of drapo repo ?

Reply:
This is the url of the class in that repo that have this info
https://github.com/spadrapo/drapo/blob/master/src/Middleware/Drapo/ts/DrapoFunctionHandler.ts